src/redux/coinlist.js

```diff
@@ -0,0 +1,90 @@
+/* @flow */
+import type {
+  Dispatch as ReduxDispatch,
+} from 'redux';
+import * as _ from 'lodash';
+
+import type { CoinList } from '../Types';
```

Comment:
not sure if you need ```type``` here. I kinda liked it because it gives more clear signal this is a type import. But pretty sure removing it will work too